src/API/Site/Controllers/Ads/AccountController.php

```diff
@@ -86,6 +86,39 @@ public function register_routes(): void {
 				],
 			]
 		);
+
+		$this->register_route(
+			'ads/accepted-customer-data-terms',
+			[
+				[
+					'methods'             => TransportMethods::READABLE,
+					'callback'            => $this->get_accepted_customer_data_terms_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+				],
+			]
+		);
+
+		$this->register_route(
+			'ads/update-enhanced-conversion-status',
+			[
+				[
+					'methods'             => TransportMethods::EDITABLE,
+					'callback'            => $this->update_enhanced_ads_conversion_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+				],
+			]
+		);
+
+		$this->register_route(
+			'ads/allow-enhance-conversions',
+			[
+				[
+					'methods'             => TransportMethods::READABLE,
+					'callback'            => $this->get_enhanced_conversion_status_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+				],
+			]
+		);
```

Comment:
Just spotted that we can consolidate these two into a single endpoint that can be used for reading and updating the status of this option. Here's an example, but will require changes to all the places where we are using these endpoints

```suggestion
		$this->register_route(
			'ads/enhanced-conversion-status',
			[
				[
					'methods'             => TransportMethods::EDITABLE,
					'callback'            => $this->update_enhanced_ads_conversion_callback(),
					'permission_callback' => $this->get_permission_callback(),
				],
				[
					'methods'             => TransportMethods::READABLE,
					'callback'            => $this->get_enhanced_conversion_status_callback(),
					'permission_callback' => $this->get_permission_callback(),
				],
			]
		);
```

Reply:
Good catch @joemcgill . Updated!